### ISSUE TITLE
hls(.m3u8) as video input?

### ISSUE THREAD

**Issue by flyingmrwang** [NVIDIA-AI-IOT/deepstream_tlt_apps#34]
Hi, does Deepstream support m3u8 live stream video as input? If yes, how to build the pipeline to capture and decode input?

**Comment by mchi-zg** [comment]
yes. refer to https://forums.developer.nvidia.com/t/unable-to-consume-hls-stream-from-kinesis-feed/128284/3?u=mchi
